### PR TITLE
OGM-993 Modules for Infinispan should use the Infinispan community modules rather than the WildFly ones

### PR DESCRIPTION
**Pull request by Sanne**
https://hibernate.atlassian.net/browse/OGM-993

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Nice; Should we add a note how users would update the slot OGM depends on? Or just say "OGM uses version x.y.z, please add the right module ZIP for that version"?

**Comment by Sanne** [comment]
Were would you have that note? BTW not sure if you noticed but I'm having it link directly to the right zip.

**Comment by gunnarmorling** [comment]
> Were would you have that note?

In the ref guide, within the section you added.

> I'm having it link directly to the right zip.

Yes, what I mean is, say ISPN 8.3 is out, and people want to use that. So
they'd have to update the referenced slot in the hibernate-ogm-infinispan
module.xml, too, IIUC. Or, we'd just say that we only support that single
version mentioned in the ref guide and that's it.

**Comment by Sanne** [comment]
> Or, we'd just say that we only support that single
version mentioned in the ref guide and that's it.

I'd avoid to mention "support" but I'll try to clarify :)

**Comment by Sanne** [comment]
pushed a new version with more verbose docs

File: documentation/manual/src/main/asciidoc/en-US/modules/configuration.asciidoc [hunk 1]
```diff
@@ -403,3 +403,22 @@ The Hibernate Search documentation explains the details of downloading and deplo
...
+so a micro version update should be safe but is untested.
+
+In case you want to experiment with a more significant version upgrade, you will need to edit the modules of Hibernate OGM:
+the module identifiers are hardcoded in the XML files representing the module.
+
+Download the Infinispan modules pack for WildFly 10 from here:
+
+ * https://repo1.maven.org/maven2/org/infinispan/infinispan-as-embedded-modules/{infinispan-version}/infinispan-as-embedded-modules-{infinispan-version}.zip[Infinispan WildFly modules version {infinispan-version} from the Maven repository]
+
+Then similarly to what you did with the Hibernate OGM mozules zip, unpack this one too in your `modules` directory within the application server.
```
Comment (DavideD) [inline documentation/manual/src/main/asciidoc/en-US/modules/configuration.asciidoc]: mo**z**ules

Comment (Sanne) [inline documentation/manual/src/main/asciidoc/en-US/modules/configuration.asciidoc]: thanks! fixed

**Comment by Sanne** [comment]
added one more commit, for the depending Infinispan upgrade to 8.2 :
https://hibernate.atlassian.net/browse/OGM-994

**Comment by gunnarmorling** [comment]
Nice change! Merge yourself if you like? Otherwise I'll do it tomorrow. Thanks!

**Comment by Sanne** [comment]
great, thanks merged.

**Pull request merged**